certified-connectors/InEight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3050 @@
+            "description": "Tenant Prefix",
+            "in": "header",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "Ocp-Apim-Subscription-Key",
+            "x-ms-summary": "This field contains Tenant Subscription key",
+            "description": "Subscription Key",
+            "in": "header",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "DailyPlanId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "DailyPlanId"
+                },
+                "PlanStatusCode": {
+                  "type": "string",
+                  "description": "PlanStatusCode"
+                },
+                "ModifiedById": {
+                  "type": "string",
+                  "description": "ModifiedById"
+                },
+                "ModifiedDate": {
+                  "type": "string",
+                  "description": "ModifiedDate"
+                },
+                "DtModifiedDate": {
+                  "type": "string",
+                  "description": "DtModifiedDate"
+                },
+                "ReleasedDate": {
+                  "type": "string",
+                  "description": "ReleasedDate"
+                },
+                "DtReleasedDate": {
+                  "type": "string",
+                  "description": "DtReleasedDate"
+                },
+                "ReleasedById": {
+                  "type": "string",
+                  "description": "ReleasedById"
+                },
+                "Errors": {
+                  "type": "array",
+                  "items": {
+                    "type": "string"
+                  },
+                  "description": "Errors"
+                }
+              }
+            }
+          }
+        ],
+        "consumes": [
+          "application/json",
+          "text/plain"
+        ],
+        "produces": [
+          "application/json",
+          "text/plain"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          }
+        }
+      }
+    },
+    "/Employees/Import": {
+      "post": {
+        "description": "The Employees Integration allows InEight customers to provide a list of all employee resources  for their entire organization, and to maintain the list as employees are hired, terminated, or  details of their employment change.<br/><br/>Within the application, Employee records are  found in the Account level menu options under ''Master data libraries'' > ''Operational resources''.",
+        "operationId": "Employees_Import",
+        "summary": "Employees_Import",
```

Comment:
@InEightDevOps The summary value exposes to the public. And all the summary and description values should be free from grammatical and spelling mistakes. 

Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)